packages/flutter/test/widgets/heroes_test.dart

```diff
@@ -1448,4 +1448,28 @@ void main() {
     ));
     expect(find.text('two'), findsOneWidget);
   });
+
+  testWidgets('Hero within a Hero, throws', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: Material(
+          child: Hero(tag: 'a', child: Hero(tag: 'b', child: Text('Child of a Hero'))),
```

Comment:
Please format flutter-style:

```suggestion
          child: Hero(
            tag: 'a', 
            child: Hero(
              tag: 'b', 
              child: Text('Child of a Hero'),
            ),
          ),
```